File: eRestoran.Model/Restoran.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eRestoran.Model
{
    public class Restoran
    {
        public int RestoranId { get; set; }
        public string? NazivRestorana { get; set; }
        public string? Email { get; set; }
        public string? Telefon { get; set; }
        public string? Adresa { get; set; }
        public int? GradId { get; set; }
        public virtual Grad? Grad { get; set; }
    }
}

File: UI/erestoran_admin/lib/models/restoran.dart
import 'package:json_annotation/json_annotation.dart';

part 'restoran.g.dart';

@JsonSerializable()
class Restoran {
  int? restoranId;
  String? nazivRestorana;
  String? adresa;
  String? email;
  String? telefon;
  int? gradId;
  Restoran(this.restoranId, this.nazivRestorana, this.adresa, this.email,
      this.telefon, this.gradId);

  factory Restoran.fromJson(Map<String, dynamic> json) =>
      _$RestoranFromJson(json);

  Map<String, dynamic> toJson() => _$RestoranToJson(this);
}
